lib/Models/Cesium.ts

```diff
@@ -114,9 +114,12 @@ export default class Cesium extends GlobeOrMap {
   private readonly _disposeSplitterReaction: () => void;
 
   private _createImageryLayer: (
-    ip: ImageryProvider
-  ) => ImageryLayer = createTransformer((ip: ImageryProvider) => {
-    return new ImageryLayer(ip);
+    ip: ImageryProvider,
+    clippingRectangle: Rectangle | undefined
+  ) => ImageryLayer = computedFn((ip, clippingRectangle) => {
```

Comment:
Why the move from createTransformer to computedFunction? I'm not super confident that I understand the difference between the two, but I thought createTransformer memoized functions that were a pure function of their arguments, whereas a computedFn memoized functions that are a function of props and observable state.

Reply:
I am also not entirely sure what the difference is between them. One difference is that `computedFn` can take multiple arguments (as long as the number of arguments is always the same) and those arguments are allowed to be undefined.

`computedFn` is newer than `createTransformer` and the creator of mobx seems to recommend `computedFn` where it's possible to use it. The only thing I can think of that would make it not possible to use it is if you're using `createTransformer`'s destructor option, but I'm sure there are some other differences too.